sdcm/utils/cdc/options.py

```diff
@@ -11,7 +11,7 @@
 CDC_SETTINGS_NAMES_VALUES = {
     "enabled": [True, False],
     "delta": ["full", "keys"],
-    "preimage": ["full", "true", "false"],
+    "preimage": ["full", True, False],
```

Comment:
and what happens if it would select "full" ? would it work ?

I think you need to convert it again via the same function, after altering/randomizing it